packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1370,10 +1370,10 @@ export default class Meetings extends WebexPlugin {
 
         if (Object.keys(meetingsCollection).length > 0) {
           // Some time the mercury event is missed after mercury reconnect
-          // if sync returns no locus then clear all the meetings
+          // if sync returns invalid locus events, then clear them and keep the rest
```

Comment:
It would be better to use the term "meetings" instead of "events". Also, the code is removing the meetings from meetingCollection not from the list that sync returns, so I think the comment would be better like this: "remove any Locus meetings that are not returned by Locus in the sync"